base/logger_audit.go

```diff
@@ -131,12 +132,28 @@ type AuditLogger struct {
 	config AuditLoggerConfig
 }
 
+func (l *AuditLogger) getAuditLoggerConfig() *AuditLoggerConfig {
+	c := AuditLoggerConfig{}
+	if l != nil {
+		// Copy config struct to avoid mutating running config
```

Comment:
Does this help a lot given that all the member of AuditLoggerConfig are pointers and this is a shallow copy?

Reply:
Doesn't seem nessesary since there's no runtime configuration beyond the embedded FileLoggerConfig (yet), but I think I'd rather just keep this here in case we do add runtime-configurable options to `auditLogger`. Follows the same pattern as console and file loggers.